scripts/jest/jest-cli.js

```diff
@@ -104,6 +104,12 @@ const argv = yargs
       type: 'boolean',
       default: false,
     },
+    version: {
```

Comment:
nit: I wonder if `reactVersion` would be a more meaningful name since DevTools also has versions?